README.md

```diff
@@ -67,6 +67,31 @@ distinguished by their unique VIN numbers:
       ]
     }
 
+## Homekit
+
+For vehicles with Homekit support, the plugin allows you to enable the feature to send a Homekit signal from the car. This is disabled by default. To enable the feature, you may use the UI (uncheck the Disable Homelink Service to enable it) or add a setting in the accessory section for the vehicle. You also must provide a latitude and longitude value for the Homekit device.
+
+    {
+      "accessories": [
+        {
+          "accessory": "Tesla",
+          "name": "Model 3",
+          "vin": "5JJYCB522AB296261",
+          "username": "bobs@burgers.com",
+          "password": "bobbobbaran"
+          "disableHomelink": false,
```

Comment:
This isn't quite right, since in the code this value will be `false` by default - so it's not really disabled unless you *add* the flag in your JSON with the value of `true`. Perhaps this flag could be `enableHomeLink` instead, so that omitting it provides the default value of "off"?

Reply:
Changed to enableHomelink and set default to false.